Can not create a new product in Perkstore with Postgresql DB

**Steps to reproduce**

Steps to reproduce the behavior:
1- Using a Postgresql database
2- Connect and try to create a new Product
3- An error occur : 
``` Caused by: org.postgresql.util.PSQLException: ERROR: operator does not exist: integer = boolean
```


If applicable, add screenshots to help explain your problem.

**(optional) Affected version**

Indicate the version where the bug is observed.

**Current behavior**

- Products could not be created
- Perkstore widget in SNAPSHOT page is not displayed
- 
**Expected behavior**

Product should be created successfully

**(optional) Affected environments**  
Please complete the following information: 
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]
 - Device: [e.g. iPhone6]

--- a/perk-store-services/src/main/resources/db/changelog/perkstore-rdbms.db.changelog-1.0.0.xml
+++ b/perk-store-services/src/main/resources/db/changelog/perkstore-rdbms.db.changelog-1.0.0.xml
@@ -179,4 +179,12 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       </column>
     </addColumn>
   </changeSet>
+
+  <changeSet author="perkstore" id="1.0.0-11">
+    <sql dbms="postgresql" >
+      ALTER TABLE ADDONS_PERKSTORE_PRODUCT ALTER COLUMN DELETED DROP DEFAULT;
+      ALTER TABLE ADDONS_PERKSTORE_PRODUCT ALTER DELETED TYPE bool USING DELETED::boolean;
+      ALTER TABLE ADDONS_PERKSTORE_PRODUCT ALTER COLUMN DELETED SET DEFAULT FALSE;
+    </sql>
+  </changeSet>
 </databaseChangeLog>

